front/lib/plans/workspace_usage.ts

```diff
@@ -0,0 +1,24 @@
+import { Op } from "sequelize";
+
+import { Membership, Workspace } from "@app/lib/models/workspace";
+
+export async function countActiveSeatsInWorkspace(
+  workspaceId: string
+): Promise<number> {
+  const workspace = await Workspace.findOne({
+    where: {
+      sId: workspaceId,
+    },
+  });
+  if (!workspace) {
+    throw new Error(`Workspace not found for sId: ${workspaceId}`);
+  }
+  return await Membership.count({
+    where: {
+      workspaceId: workspace.id,
+      role: {
+        [Op.in]: ["user", "builder", "admin"],
```

Comment:
nit: because we have no type-safety here, would it be preferable to to `Op.not: "none"` ? that way if we add more roles then we don't accidentally forget to count them.

To be fair, my actual preferred solution would be to make `RoleType` be a string union derived from a `const` array, so we could have type safety

Reply:
Replaced by `[Op.notIn]: ["none", "revoked"],`